app/containers/FarmingPage/index.js

```diff
@@ -47,16 +47,33 @@ export function FarmingPage(props) {
   const [farmingData, setFarmingData] = useState([]);
   const [farmingModal, setFarmingModal] = useState(false);
   const [farmingFee, setFarmingFee] = useState(10);
+  const [initialLoad, setInitialLoad] = useState(true)
 
   useEffect(() => {
     refreshTokenStaked();
   }, [wallet]);
 
+  useEffect(() => {
+    const RGPfarmingFee = async () => {
+      if (wallet.signer !== 'signer') {
+        const masterChef = await masterChefContract();
+        const minFarmingFee = await masterChef.farmingFee();
+        const fee = Web3.utils.fromWei(minFarmingFee.toString());
+        setFarmingFee(fee);
+        props.changeRGPFarmingFee({
+          fee,
+        });
+      }
+    };
+    RGPfarmingFee();
+    initialLoad ? setFarmingModal(true):setFarmingModal(false)
```

Comment:
Kindly put this in a function and call the function.

Reply:
resolved